### PR TITLE
Feature/Audio Source AM/FM/XM/DAB 

### PR DESCRIPTION
**Pull request by LitvinenkoIra**
ATF Test Scripts to check  #[2338](https://github.com/smartdevicelink/sdl_core/issues/2338) 
Note: Implementation is based on #[1798](https://github.com/smartdevicelink/sdl_core/issues/1798), so this PR should be merged after #1909 

### Summary
Scripts for feature https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0182-audio-source-am-fm-xm.md

### ATF version
[Develop branch](https://github.com/smartdevicelink/sdl_atf/tree/develop)

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by yang1070** [comment]
I just run the test scripts under folder /RC/AUDIO_LIGHT_HMI_SETTINGS/
There are some errors, please investigate

**Comment by yang1070** [comment]
@LitvinenkoIra Ford has reviewed, run all the test scripts successfully, and approve the change. It is ready for Livio to review.

File: test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/SetInteriorVehicleData/017_Change_audio_source_from_any_type_except_MOBILE_APP_keepContext_true.lua [hunk 1]
```diff
@@ -34,7 +34,10 @@ local audioSources = {
   "USB2",
   "LINE_IN",
   "IPOD",
-  "RADIO_TUNER"
+  "AM",
+  "FM",
+  "XM",
+  "DAB"
 }
```
Comment (jacobkeeler) [inline test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/SetInteriorVehicleData/017_Change_audio_source_from_any_type_except_MOBILE_APP_keepContext_true.lua]: Use `common.audioSources`

Comment (GetmanetsIrina) [reply test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/SetInteriorVehicleData/017_Change_audio_source_from_any_type_except_MOBILE_APP_keepContext_true.lua]: @jacobkeeler , common.audioSources contains "MOBILE_APP" value, and this script tests all types except "MOBILE_APP", so here array is defined localy without "MOBILE_APP".

**Pull request merged**